clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/BlobResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.davinci.blob;
+
+import java.util.List;
+import java.util.Objects;
+
+
+public class BlobResponse {
+  private boolean isError;
+
+  private String message;
+
+  private List<String> partitionURLs;
+
+  public void setError(boolean error) {
+    this.isError = error;
+  }
+
+  public boolean isError() {
+    return this.isError;
+  }
+
+  public void setMessage(String message) {
+    this.message = message;
+  }
+
+  public String getMessage() {
+    return this.message;
+  }
+
+  public void setPartitionValues(List<String> partitionURLs) {
```

Comment:
the method name might be better if it matches the field it is setting to. I.e. `setPartitionURLs`

Reply:
its now called setDiscoveryResult